app/src/ui/autocompletion/autocompleting-text-input.tsx

```diff
@@ -273,9 +273,9 @@ export abstract class AutocompletingTextInput<
           rowCount={items.length}
           rowHeight={RowHeight}
           rowId={this.getRowId}
-          selectedRows={[selectedRow]}
+          selectedRows={selectedRows}
           rowRenderer={this.renderItem}
-          scrollToRow={selectedRow}
+          scrollToRow={selectedRows.at(0)}
```

Comment:
On list component `scrollToRow` expects `undefined` or a valid number (> -1) here too. Looks we don't see any negative consequence (assuming the react Grid component must have good sanitization) of the negative 1.. but using `at` will return `undefined` and will prevent our code from attempting to scroll. 

```  /** The row that should be scrolled to when the list is rendered. */
  readonly scrollToRow?: number